syntax = "proto3";
package deploy;




import "model/easy_flow/deploy_ret.proto";

/*
List请求
*/
message ListRequest {
    /*
    任务id,使用分号分隔
    */
    string taskIdList = 1 ;
}

/*
List返回
*/
message ListResponse {
    /*
    错误码
    */
    int32 code = 1 ;
    /*
    消息
    */
    string msg = 2 ;
    /*
    返回数据
    */
    repeated easy_flow.DeployRet data = 3 ;
}

/*
ListApi返回
*/
message ListResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    deploy.ListResponse data = 4 ;
}

